crates/cargo-wdk/src/actions/build/mod.rs

```diff
@@ -44,7 +47,7 @@ pub struct BuildActionParams<'a> {
 pub struct BuildAction<'a> {
     working_dir: PathBuf,
     profile: Option<&'a Profile>,
-    target_arch: TargetArch,
+    target_arch: Option<&'a CpuArchitecture>,
```

Comment:
```suggestion
    target_arch: Option<CpuArchitecture>,
```

`CpuArchitecture` is a simple enum and should be passed by value

Reply:
Fixed ✅